meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config.bb

```diff
@@ -10,13 +10,16 @@ SRC_URI = " \
     "
 S = "${WORKDIR}"
 
-inherit allarch systemd
+inherit allarch systemd useradd
```

Comment:
Can we rebase this commit in the initial commit and adding the relevant information in the commit log?

As well this needs a little more information on why it's needed.